text/0017-lmdb-vs-leveldb.md

```diff
@@ -0,0 +1,238 @@
+# LMDB vs. LevelDB
+
+This document compares the [Lightning Memory-mapped Database](https://symas.com/lmdb/) (LMDB) key-value storage engine to the [LevelDB](https://github.com/google/leveldb) key-value storage engine.
+
+## Meta
+
+### Project Structure
+
+#### LMDB
+
+LMDB is developed by <a href="https://symas.com/">Symas Corporation</a>, a small "technical support firm" whose business model is to provide technical support for the open-source software it develops. Its primary product is OpenLDAP, and it originally developed LMDB for use in that server software, which continues to use it. It provides paid technical support for LMDB.
+
+The LMDB code is stored in OpenLDAP's repository and read-only mirrored to GitHub. Issues and patches are accepted upstream, not in the GitHub repository. LMDB doesn't appear to have an active community beyond the core developers.
+
+#### LevelDB
+
+LevelDB was "written at Google" (according to its <a href="https://github.com/google/leveldb">GitHub project page</a>) and "inspired by <a href="https://en.wikipedia.org/wiki/Bigtable">BigTable</a>" (according to its <a href="https://en.wikipedia.org/wiki/LevelDB">Wikipedia entry</a><sup>[1](#footnote-1)</sup>), although it doesn't share code with that proprietary storage system. It was originally developed, among other reasons, to back IndexedDB in Google Chrome, which continues to use it for that purpose.
```

Comment:
nit: the capitalization of "bigtable" differs in the link and the anchor.  Intentional?

Reply:
Unintentional! It looks like the project capitalizes its name "Bigtable," so I've fixed the capitalization in 811db34.